### ISSUE TITLE
Issue with Trip Planning Legs

### ISSUE THREAD

**Issue by TNWolf** [CUTR-at-USF/usf-mobullity#14]
When planning a trip, gaps appear between steps on the map with the legs that show the user what path to take.

![screenshot 2014-10-08 12 37 46](https://cloud.githubusercontent.com/assets/5204472/4581841/6385bade-4fe3-11e4-9505-055b112ab00f.png)

**Comment by jmfield2** [comment]
I think PR #13 may fix this -- related to internal bus/bike stop linking code

**Comment by TNWolf** [comment]
Still having a similar issue after pull request #13 was merged.

**Comment by barbeau** [comment]
@jailby Can you please take a look at this issue and see if you can reproduce it on the public facing site?  It may be fixed now.

**Comment by jailby** [comment]
I think it is.

I never saw this issue while testing and I have tested the trip planner quite a lot.
I also tried to reproduce this specific trip but the bus in the example was not present on my test. I believe it's a HART bus and might not be available for some reasons (wrong times etc...).
Therefore, I didn't notice anything wrong. 

There is often a gap between bus stop and actual sidewalk but it's normal. The sidewalk is not always connected to the Bus stop on OSM. (The gap is small)
